test/integration/three_d_secure_ft_tests.js

```diff
@@ -60,6 +60,8 @@ describe('chargeTests', function () {
         getChargeRequest(app, cookieValue, chargeId, '/3ds_required_out')
           .expect(200)
           .expect(function (res) {
+            expect(res.header['set-cookie'][0].startsWith('frontend_state=')).to.be.true
+            expect(res.header['set-cookie'][0].length).to.be.greaterThan(100)
```

Comment:
I think we decided that it wasn’t worth checking we’re setting the cookie here. We just want to check on the 3DS required in pages that we definitely don’t.

Reply:
This check was in a few places, not only lines 63/64. I have removed the other reference too.